api/v1alpha1/project_config_types.go

```diff
@@ -13,13 +24,42 @@ type ProjectConfig struct {
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 	// Spec describes the configuration of a Project.
 	Spec ProjectConfigSpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
+	// Status describes the current status of a Project Config.
+	Status ProjectConfigStatus `json:"status,omitempty" protobuf:"bytes,3,opt,name=status"`
+}
+
+func (p *ProjectConfig) GetStatus() *ProjectConfigStatus {
+	return &p.Status
 }
 
 // ProjectConfigSpec describes the configuration of a Project.
 type ProjectConfigSpec struct {
 	// PromotionPolicies defines policies governing the promotion of Freight to
 	// specific Stages within the Project.
 	PromotionPolicies []PromotionPolicy `json:"promotionPolicies,omitempty" protobuf:"bytes,1,rep,name=promotionPolicies"`
+	// WebhookReceiverConfigs defines the webhook receivers for the project config.
+	WebhookReceiverConfigs []WebhookReceiverConfig `json:"webhookReceivers,omitempty" protobuf:"bytes,2,rep,name=receivers"`
+}
+
+type ProjectConfigStatus struct {
```

Comment:
Add comment... especially important in *_type.go files because these comments all make their way into the CRD definitions and documentation that's automatically extracted from those.